### PR TITLE
chore(console-lite): add new stepper component and change deal ticket…

### LINKED ISSUE #529: ConsoleLite Design Update: Stepper Component
ACs:
- [ ] It should match the design
- [ ] It should work on dark/light mode
- [ ] It should display horizontally on mobile
- [ ] It should work on screen readers
- [ ] It should have an option to display the multicoloured animated circle backgrounds

### PR DESCRIPTION
**Pull request by elmar-vega**
… steps

# Related issues 🔗

Closes #529 

# Description ℹ️

Updated the new stepper component to match the new design. Removed old implementation using MUI.

# Demo 📺

Screenshot 2022-07-18 at 10 53 08
Screenshot 2022-07-18 at 10 52 52

# Technical 👨‍🔧

- Removed MUI from project
- Added a11y optimised stepper component
- Added tests

### CODE REVIEW

File: apps/simple-trading-app/src/app/components/stepper/counter.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+interface CounterProps {
+  label: string;
+  className?: string;
+  isActive: boolean;
+}
+
+export const Counter = ({ label, isActive, className }: CounterProps) => {
+  const classes = classNames(
+    'rounded-full relative border-black dark:border-white bg-white dark:bg-black text-black dark:text-white aspect-square clip-path-rounded flex items-center justify-center text-center',
```
Comment (sam-keen) [inline apps/simple-trading-app/src/app/components/stepper/counter.tsx:13]: Chore (non-blocking): I've found pulling these long classname strings into multiple lines, grouped by type, makes them a bit more readable

File: apps/simple-trading-app/src/app/components/stepper/stepper.tsx [hunk 2]
```diff
@@ -1,93 +1,143 @@
...
-                <div>
-                  <Button
-                    variant="secondary"
-                    onClick={handleNext}
-                    disabled={step.disabled}
+              <div className="flex relative md:pt-16">
+                {!isFirstStep ? (
+                  <div
+                    aria-hidden
+                    className="flex-auto	absolute top-[20px] -left-1/2 right-1/2 md:-top-1/2 md:bottom-1/2 md:left-[29.5px] md:right-auto"
```
Comment (sam-keen) [inline apps/simple-trading-app/src/app/components/stepper/stepper.tsx:65]: Chore (non-blocking): Remove unnecessary text gap here and on line 78

**Review by sam-keen — COMMENTED** [review]
Couple of small things, looks good :)

**Review by macqbat — APPROVED** [review]

**Review by dexturr — APPROVED** [review]

File: apps/simple-trading-app/src/app/components/stepper/counter.tsx (hunk 1 above)
Comment (elmar-vega) [reply apps/simple-trading-app/src/app/components/stepper/counter.tsx]: Done

File: apps/simple-trading-app/src/app/components/stepper/stepper.tsx (hunk 2 above)
Comment (elmar-vega) [reply apps/simple-trading-app/src/app/components/stepper/stepper.tsx]: Done

**Pull request merged**